planning/obstacle_avoidance_planner/src/node.cpp

```diff
@@ -106,6 +111,12 @@ ObstacleAvoidancePlanner::ObstacleAvoidancePlanner(const rclcpp::NodeOptions & n
 
     // parameters for trajectory
     traj_param_ = TrajectoryParam(this);
+
+    // transform
+    rclcpp::Clock::SharedPtr clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+    map_odom_tf_buffer_ = std::make_shared<tf2_ros::Buffer>(clock);
```

Comment:
You can get the clock that is used by this node instead of creating a new one with `this->get_clock()`.

Reply:
> You can get the clock that is used by this node instead of creating a new one with `this->get_clock()`.

got it. thanks.